api/controller/dbController.js

```diff
@@ -0,0 +1,83 @@
+import { query } from '../../db/db'
+import { 
+  artistQueries, setlistQueries, songQueries, setlistSongQueries
+  } from '../../db/queries'
+
+  export class dbController {
+    
+    addAllInfo = async ({apiResult,externalId,artistName}) => {
+      const artistId = await this.addedArtist({externalId,artistName})
+      let listOfShows = apiResult.setlist
+      for (const concert of listOfShows){
+        let concertId = await this.addSetlist({artistId, concertExternalId:concert.id, updatedAt:concert.lastUpdated, eventDate:concert.eventDate})
+        let concertSets = concert.sets.set
```

Comment:
This one is weird. `concert.sets.set == concertSets`? I would assume `concert.sets = concertSets` or `concert.sets.set == set`. I might need to see the data how it comes in to figure this one out.

Reply:
I changed this one to sets = setlist.sets.set. This one is a bit of a headache from the API data. For some reason each concert date has a 'sets' object that contains one object 'set' which for some reason has all of the set objects. Maybe they have a good reason for it but I agree that it's counterintuitive.

Reply:
Oof yeah that API data is a bit confusing. I think that renaming it to `set` is a big improvement though!